.mvn/srcdeps.yaml

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright 2014-2016 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# `.mvn/srcdeps.yaml` - the srcdeps configuration file
+
+# `configModelVersion` exists just in case we'll have to change the schema of srcdeps.yaml in the future.
+#
+# Required, no default, only 1.0 supported at the moment
```

Comment:
BTW, the CI is passing here now is a proof that the -SRC- version in parent works. Although it is great that it works, I think it will be more practical to release `hawkular-parent` 50 now and upgrade it here straight away in this PR. WDYT? Could you please eventually release hawkular-parent now?